src/settings/mod.rs

```diff
@@ -333,21 +321,21 @@ fn resolve_codes<'a>(specs: impl IntoIterator<Item = RuleCodeSpec<'a>>) -> FxHas
             SuffixLength::Three,
             SuffixLength::Four,
         ] {
```

Comment:
How does the specificity work between `PL` and (e.g.) `PLC`? What would `--select PLC --ignore PL` do?

Reply:
Very good question! ... thanks that was indeed broken ... I just force pushed a fix that made this work as expected (the more specific selector takes precedence).

Please note that all the special logic that was necessary to implement the `PL` code will be removed in my followup PR that changes how `RuleSelector` works (which is also why I didn't implement special test cases for PL since these will very shortly be redundant since PL will be just a prefix like any other prefix that's configured via the `#[prefix]` macro attribute).

Reply:
Is that alright?

Reply:
Ah, ok, that's reasonable. I see the special-casing now. It effectively treats the `R` in `PLR0` as a numerical piece, IIUC.

Reply:
Yes exactly that's the fix I just implemented.

And my follow up PR will change the rule selector parsing to firstly parse the configured `#[prefix]`es so `PLR0` will be split to `PL` (= `Pylint`), `R0`  ... which will let us remove that special casing :)